src/OrleansCodeGenerator/RoslynCodeGenerator.cs

```diff
@@ -484,6 +480,36 @@ private static void RecordType(Type type, Module module, Assembly targetAssembly
         }
 
         /// <summary>
+        /// Get types which have corresponding generated classes.
+        /// </summary>
+        /// <returns>Types which have corresponding generated classes marked.</returns>
+        private static HashSet<Type> GetTypesWithGeneratedSupportClasses()
+        {
+            // Get assemblies which contain generated code.
+            var all =
+                AppDomain.CurrentDomain.GetAssemblies()
+                    .Where(_ => _.GetCustomAttribute<GeneratedCodeAttribute>() != null)
+                    .SelectMany(_ => _.DefinedTypes);
```

Comment:
I used GetCustomAttribute, not GetCustomAttribute**s**. I can give those args names if you prefer.